.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Weekly Thursday 5AM build
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 5 * * 4'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
```

Comment:
I wonder whether for all these small package with a single CI build we should just use 3.x then don't need to update in future?

Reply:
Did that for my stuff but then things broke (or very slow) when `3.x` became 3.9 but numpy/scipy didn't catch up yet. 🤷

Reply:
But I guess should be ok for this one...